Documentation/LegacyAI/LegacyWorkbooksConversion.md

```diff
@@ -69,4 +69,8 @@ Starting July 1 2021 you will not see your Legacy Workbooks in Azure Portal.
 You will still be able to retrieve the content of Legacy Workbooks for another 30 days (until July 31 2021) by using Azure CLI or PowerShell tools, to query `microsoft.insights/components/[name]/favorites` for the specific resource using `api-version=2015-05-01`. 
 
 ## What if I *need* to temporarily save change to a Legacy Workbook
```

Comment:
you'll need to remove this feature flag section, it no longer applies after your code changes